fulp_modules/features/halloween/2021/shovel_knight_outfits/propeller_knight.dm

```diff
@@ -0,0 +1,35 @@
+/**
+ * Propeller Knight
+ * From: Shovel knight
+ * By: Sheets
+ */
+
+///Suit (Toggled)
+/obj/item/clothing/suit/hooded/costume_2021/propeller_suit
+	name = "propeller knight's suit"
+	desc = "A tight, yet comfortable green suit."
+	icon_state = "propeller_suit"
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS|FEET
+	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
+	hoodtype = /obj/item/clothing/head/hooded/costume_2021/propeller_head
+
+///Hood
+/obj/item/clothing/head/hooded/costume_2021/propeller_head
+	name = "propeller knight's helmet"
+	desc = "A reflective gold helmet with a makeshift propeller device fastened ontop."
+	icon_state = "propeller_head"
+	body_parts_covered = HEAD
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEFACIALHAIR|HIDEHAIR
+
+///Gloves
+/obj/item/clothing/gloves/costume_2021/propeller_gloves
+	name = "propeller gloves"
+	desc = "A tight yet comfortable pair of gloves."
+	icon_state = "propeller_gloves"
```

Comment:
you could make these (improvised-tier) tackling gloves if you wanted people to try to "fly" with them